examples/with-next-auth/pages/api/auth/[...nextauth].js

```diff
@@ -2,97 +2,118 @@ import NextAuth from 'next-auth'
 import Providers from 'next-auth/providers'
 
 const options = {
-  site: process.env.VERCEL_URL,
+  // @link https://next-auth.js.org/configuration/providers
   providers: [
     Providers.Email({
       // SMTP connection string or nodemailer configuration object https://nodemailer.com/
-      server: process.env.EMAIL_SERVER,
+      server: process.env.NEXTAUTH_EMAIL_SERVER,
       // Email services often only allow sending email from a valid/verified address
-      from: process.env.EMAIL_FROM,
+      from: process.env.NEXTAUTH_EMAIL_FROM,
     }),
     // When configuring oAuth providers make sure you enabling requesting
     // permission to get the users email address (required to sign in)
     Providers.Google({
-      clientId: process.env.GOOGLE_ID,
-      clientSecret: process.env.GOOGLE_SECRET,
+      clientId: process.env.NEXTAUTH_GOOGLE_ID,
+      clientSecret: process.env.NEXTAUTH_GOOGLE_SECRET,
     }),
     Providers.Facebook({
-      clientId: process.env.FACEBOOK_ID,
-      clientSecret: process.env.FACEBOOK_SECRET,
+      clientId: process.env.NEXTAUTH_FACEBOOK_ID,
+      clientSecret: process.env.NEXTAUTH_FACEBOOK_SECRET,
     }),
     Providers.Twitter({
-      clientId: process.env.TWITTER_ID,
-      clientSecret: process.env.TWITTER_SECRET,
+      clientId: process.env.NEXTAUTH_TWITTER_ID,
+      clientSecret: process.env.NEXTAUTH_TWITTER_SECRET,
     }),
     Providers.GitHub({
-      clientId: process.env.GITHUB_ID,
-      clientSecret: process.env.GITHUB_SECRET,
+      clientId: process.env.NEXTAUTH_GITHUB_ID,
+      clientSecret: process.env.NEXTAUTH_GITHUB_SECRET,
     }),
   ],
-  // The 'database' option should be a connection string or TypeORM
-  // configuration object https://typeorm.io/#/connection-options
-  //
-  // Notes:
-  // * You need to install an appropriate node_module for your database!
-  // * The email sign in provider requires a database but OAuth providers do not
-  database: process.env.DATABASE_URL,
 
+  // @link https://next-auth.js.org/configuration/databases
+  database: process.env.NEXTAUTH_DATABASE_URL,
+
+  // @link https://next-auth.js.org/configuration/options#session
   session: {
     // Use JSON Web Tokens for session instead of database sessions.
     // This option can be used with or without a database for users/accounts.
     // Note: `jwt` is automatically set to `true` if no database is specified.
-    // jwt: false,
+    //jwt: true,
     // Seconds - How long until an idle session expires and is no longer valid.
-    // maxAge: 30 * 24 * 60 * 60, // 30 days
```

Comment:
Nit pick: Removing that empty first space makes it look out of place compared to the other comments.

Reply:
Good catch, fixed 

if you really want my opinion on this: I would ditch all those comments and only refer to next-auth doc, cause you know comments change :)

A possible alternative,  use a mix of `/** */` and `//` to denote the difference between a real comment and a commented code...  

I'm glad to change either way...

Reply:
Whatever makes more sense should be used. I'm only the quality reviewer.

Reply:
And the best one :)